postgres_kernel/kernel.py

```diff
@@ -181,16 +181,24 @@ def do_execute(self, code, silent, store_history=True,
                     'ename': 'ProgrammingError', 'evalue': str(e),
                     'traceback': []}
         else:
-            if header is not None:
+            # Modified by mhoangvslev
+            self.send_response(
+                    self.iopub_socket, 'stream', {
+                        'name': 'stdout',
+                        'text': str(len(rows)) + " row(s) returned."
+                        })
+            if header is not None and len(rows) > 0:
                 self.send_response(self.iopub_socket, 'display_data', display_data(header, rows))
 
         return {'status': 'ok', 'execution_count': self.execution_count,
                 'payload': [], 'user_expressions': {}}
 
+# Modified by mhoangvslev
 def display_data(header, rows):
+    # https://bitbucket.org/astanin/python-tabulate
     d = {
         'data': {
-            'text/plain': tabulate(rows, header, tablefmt='simple'),
+            'text/latex': tabulate(rows, header, tablefmt='latex_booktabs'),
```

Comment:
Any reason to drop `text/plain` support? it seems like we can add `text/latex` without removing plain text as a possible content type.

Reply:
I tested with all sort of combination: the first must be text/latex and the second must be either latex or latex bookstab

Reply:
Gotcha. I guess what I'm saying is: Any reason we can't do this?

```suggestion
            'text/latex': tabulate(rows, header, tablefmt='latex_booktabs'),
            'text/plain': tabulate(rows, header, tablefmt='simple'),
```

Reply:
If the table is printed in plain and it has many attributes, it won't be scaled to fit the page

Reply:
@mhoangvslev I just tested your branch, and it doesn't produce output when running in the console:

![](https://trello-attachments.s3.amazonaws.com/58d428743111af1d0a20cf28/5bfdf2eb99985f018321281f/e4b00691624a411ab278511ff1a41403/capture.png)

Here is the output of the same query in the master branch:

![](https://trello-attachments.s3.amazonaws.com/58d428743111af1d0a20cf28/5bfdf30be1abcd0453e7bbd9/2f7b37a062177810d506b77eff541490/capture.png)

I suspect this difference is because your branch is missing the `text/plain` content type. You can try this for yourself by running `jupyter console --kernel postgres`

Reply:
Doubtlessly, I never think about console usage.